### PR TITLE
Fix fileconfig

### PR DESCRIPTION
**Pull request by jsdpag**
Basic edits to cbmex.cpp to get 'fileconfig' function to run properly.

### CODE REVIEW

File: cbmex/cbmex.cpp [hunk 1]
```diff
@@ -1763,22 +1763,31 @@ void OnFileConfig(
     // Do a quick look at the options just to find the instance if specified
     for (int i = 1; i < nrhs; ++i)
```
Comment (dashesy) [inline cbmex/cbmex.cpp]: please change this to

    for (int i = nFirstParam; i < nrhs; ++i)

File: cbmex/cbmex.cpp [hunk 2]
```diff
@@ -1763,22 +1763,31 @@ void OnFileConfig(
     // Do a quick look at the options just to find the instance if specified
     for (int i = 1; i < nrhs; ++i)
     {
+        
+        /* Cheat-fix. We know that prhs[ 3 ] is the 'action', by definition
+           of 'fileconfig'. However, a numeric argument will trigger an
+           error inside this for loop. To avoid this, skip input 3. There
+           is code further down that does basic type and scalar checking.
+        */
+        if  ( i == 3 )
```
Comment (dashesy) [inline cbmex/cbmex.cpp]: by other changes, you can remove this line (and the comments)

File: cbmex/cbmex.cpp [hunk 3]
```diff
@@ -1763,22 +1763,31 @@ void OnFileConfig(
...
             {
                 char errstr[128];
                 sprintf(errstr, "Parameter %d is invalid", i);
-                PrintHelp(CBMEX_FUNCTION_COMMENT, true, errstr);
+                PrintHelp(CBMEX_FUNCTION_FILECONFIG, true, errstr);
             }
             if (_strcmpi(cmdstr, "instance") == 0)
             {
                 param = PARAM_INSTANCE;
             }
```
Comment (dashesy) [inline cbmex/cbmex.cpp]: add this block:

            else if (_strcmpi(cmdstr, "option") == 0)
            {
                param = PARAM_OPTION;
            }
            else
            {
                param = PARAM_NONE; // Just something valid but instance
            }

File: cbmex/cbmex.cpp [hunk 4]
```diff
@@ -1763,22 +1763,31 @@ void OnFileConfig(
...
             else
             {
-                param = PARAM_OPTION; // Just something valid but instance
+                param = PARAM_NONE; // Just something valid but instance
```
Comment (dashesy) [inline cbmex/cbmex.cpp]: revert to 

    param = PARAM_NONE

**Comment by jsdpag** [comment]
Okay, I did those changes and pushed a new commit into my fork. Will these automatically be included in the pull request?

**Comment by dashesy** [comment]
Yes, any time you push a commit to a branch for a PR it will be updates. Thanks

**Pull request merged**